app/src/main/java/com/github/wnder/MainActivity.java

```diff
@@ -146,6 +144,7 @@ private void openUploadActivity() {
      */
     private void openPreviewActivity() {
         Intent intent = new Intent(this, GuessPreviewActivity.class);
+        intent.putExtra(GuessLocationActivity.EXTRA_DISTANCE, GlobalUser.getUser().getRadius());
```

Comment:
If we can get the radius directly from the global user, it should not be passed with an extra.

Reply:
Modified!